docs_source_files/content/introduction/on_test_automation.de.md

```diff
@@ -370,13 +370,13 @@ even if Larry decides next week that he no longer likes Ruby-on-Rails
 and decides to re-implement the entire site
 in the latest Haskell bindings with a Fortran front-end.
 
-Your page objects will require some small maintenance
-in order to conform to the site redesign,
```

Comment:
I believe the paragraph was already correct, what is the motivation to edit it?

Reply:
To maintain the conciseness I have replaced "in order to" with 'to'.

Reverted.